### DISCUSSION TITLE
Power distro Design

### DISCUSSION THREAD

**Discussion by conner-replogle** [Comet-Robotics/URC#7] [PCB/Power]
We need to take 12v and get it down to 5v to power multiple small electronics

[Helpful Video](https://www.youtube.com/watch?v=AmfLhT5SntE&t=109s)
[selecting parts](https://youtu.be/FqT_Ofd54fo?si=F5Y4P2HFs_-VEbAx)

Need to come up with specs Vin Vout Amps etc

**Comment by conner-replogle** [comment]
@ElDablo123 we need to find a way to power the bby pc off rover batteries

**Comment by Lucas-Igl** [comment]
Power step down/up circuit generator:

https://webench.ti.com/power-designer/switching-regulator/select

**Comment by PondDuck777** [comment]
Team Mountaineers Power DIstro PCB FIle
https://github.com/team-mountaineers-urc-2024/electronics/blob/main/PCB%20Files/Power%20Distribution%20Board/Power%20Distribution%20Board.kicad_pcb

**Comment by Lucas-Igl** [comment]
I finished doing the Power Board PCB (other than mounting points, need to know how it should be mounted). It would probably be good for someone to check the design to make sure the PCB should work right. Also if anyone needs the documents I used to determine the circuits I have posted them here as well. 

[SRP-Power Distro.zip](https://github.com/user-attachments/files/17334047/SRP-Power.Distro.zip)

[9V Out.pdf](https://github.com/user-attachments/files/17334034/9V.Out.pdf)
[12V Out.pdf](https://github.com/user-attachments/files/17334035/12V.Out.pdf)
[3.3V Out.pdf](https://github.com/user-attachments/files/17334036/3.3V.Out.pdf)
[5V Out.pdf](https://github.com/user-attachments/files/17334037/5V.Out.pdf)

**Comment by Lucas-Igl** [comment]
Update for 19v
[SRP-Power Distro 19v.zip](https://github.com/user-attachments/files/17499508/SRP-Power.Distro.19v.zip)

**Reply by pablo-collantes** [reply]
Here are some questions, comments, and ideas.

Schematic:
* What do you plan to power from each voltage and what current do you need? Will you be doing any sensitive analog measurement? I would guess 3.3V and 5V are for logic and 12V and 19V are for motors or servos.
* What is the reasoning behind C17 and C18?
* Why are C10 and C19 in series? If this is to get a higher voltage rating, it would probably be better to find a single capacitor that can handle the voltage.
* You can never have too many blinky LEDs! Maybe add one to each output so you can easily tell if the board is working.

PCB:
* Use a ground plain.
* The traces should be much wider. There are a variety of online calculators, but you probably want something on the order of 1-2mm wide for anything carrying much current.
* Minimize loop area and avoid vias in the high frequency inductor loops.
* Have power channels support at least 5 amps to sustain device current draw
* Phil's Lab has a great switching regulator video with more details:
